docs/src/routes.tsx

```diff
@@ -94,6 +98,10 @@ const Router = () => (
           />,
         ]}
         <DocsLayout exact path="/accessibility" component={Accessibility} />
+        <DocsLayout exact path="/accessibility-behaviors" component={AccessibilityBehaviors} />
+        <DocsLayout exact path="/focus-zone" component={FocusZone} />
+        <DocsLayout exact path="/focus-trap-zone" component={FocusTrapZone} />
```

Comment:
shouldn't we introduce all these sections on the same page - because, essentially, all of them are about Focus Handling, so I would see this a reasonable name for the Guides section. We might argue that the content would become pretty lengthy with these three pages being merged into one, but considering what takes the most of the space, it is the API description that we should, ideally, just refer to (have separate comment on that: https://github.com/stardust-ui/react/pull/1166/files#r273023370). Then, with this part being removed from guides content, we might just merge everything and provide client with much more consistent and consecutive set of instructions.

What do you think?

Reply:
as we discussed, I'll create a TODO for auto-generating props for docs in guides.
Also, will add links to Github's types files